2-WebApp-graph-user/2-5-HybridFlow/wwwroot/js/site.js

```diff
@@ -0,0 +1,24 @@
+﻿const msalInstance = new msal.PublicClientApplication({
+    auth: {
+        clientId: '[YOUR CLIENT ID]',
+        redirectUri: 'https://localhost:7089/',
```

Comment:
does the current setup instructions add this redirect to the sample app? or should this be documented. Also there may be app creation scripts that may need edits to include this redirect URI

Reply:
New changes to `_Layout.cshtml` should make that unnecessary if you want to take a look